bundle/src/main/java/com/adobe/acs/tools/csv_asset_importer/impl/CsvAssetImporterServlet.java

```diff
@@ -317,6 +314,24 @@ private void updateProperties(final Map<String, Column> columns,
     }
 
     /**
+     * Updates AEM System properties of the Asset.
+     * @param asset the asset to update.
+     */
+    private void updateSystemProperties(final Asset asset) {
```

Comment:
Minor issue - I'd suggest not calling these "system" properties as that has a different meaning. I'd refer to these as "standard" properties or something like that.